addons/common/XEH_postInit.sqf

```diff
@@ -133,6 +133,11 @@ if (isServer) then {
     _this lock (_this getVariable [QGVAR(lockStatus), locked _this]);
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(EjectVehicle), {
```

Comment:
```suggestion
[QGVAR(ejectVehicle), {
```

Reply:
sorry i am still testing and noticed that this still does not bring the wanted effect